compiler/src/iree/compiler/Dialect/LinalgExt/IR/TilingInterfaceImpl.cpp

```diff
@@ -181,6 +181,44 @@ LogicalResult ScatterOp::getResultTilePosition(
   return success();
 }
 
+/// Method to return the position of the result tile computed by the tiled
+/// operation.
+LogicalResult ScatterOp::getIterationDomainTileFromOperandTile(
+    OpBuilder &b, unsigned operandNumber, ArrayRef<OpFoldResult> offsets,
+    ArrayRef<OpFoldResult> sizes,
+    SmallVectorImpl<OpFoldResult> &iterDomainOffsets,
+    SmallVectorImpl<OpFoldResult> &iterDomainSizes) {
+  // Fusion with producers is not possible in general if `unique_indices` is not
+  // true as reductions along the scattered indices are not tilable in parallel.
+  if (!getUniqueIndices()) {
+    return failure();
+  }
+  // Fusion with a producer is only possible if fusing along the |input|
```

Comment:
You could fuse with index as well.....

Reply:
Good point. I updated the comment and made it TODO because it needs some verification of the offsets + sizes (+ tests) that I wasn't planning on adding here.